.hgignore

```diff
@@ -0,0 +1 @@
+.*\.pyc
```

Comment:
Small point, but I generally find it easier to use a character class to escape special characters:

.*[.]pyc$

Reply:
Ok. In a follow-up commit, I'll address that.